packages/m2m-gateway/src/services/eserviceTemplateService.ts

```diff
@@ -529,5 +530,103 @@ export function eserviceTemplateServiceBuilder(
 
       await pollEServiceTemplate(response, headers);
     },
+    async suspendEServiceTemplateVersion(
+      templateId: EServiceTemplateId,
+      versionId: EServiceTemplateVersionId,
+      { logger, headers }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.EServiceTemplateVersion> {
+      logger.info(
+        `Suspending version ${versionId} for eservice template with id ${templateId}`
+      );
+
+      const response =
+        await clients.eserviceTemplateProcessClient.suspendTemplateVersion(
+          undefined,
+          {
+            params: { templateId, templateVersionId: versionId },
+            headers,
+          }
+        );
+
+      const polledTemplate = await pollEServiceTemplateById(
+        templateId,
+        response.metadata,
+        headers
+      );
+      const version = retrieveEServiceTemplateVersionById(
+        polledTemplate,
+        unsafeBrandId(versionId)
+      );
+      return toM2MGatewayEServiceTemplateVersion(version);
+    },
+    async unsuspendEServiceTemplateVersion(
+      templateId: EServiceTemplateId,
+      versionId: EServiceTemplateVersionId,
+      { logger, headers }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.EServiceTemplateVersion> {
+      logger.info(
+        `Unsuspending version ${versionId} for eservice template with id ${templateId}`
+      );
+      const eserviceTemplate = await retrieveEServiceTemplateById(
+        headers,
+        templateId
+      );
+      const versionBefore = retrieveEServiceTemplateVersionById(
+        eserviceTemplate,
+        versionId
+      );
+      assertEServiceTemplateVersionIsSuspended(versionBefore);
```

Comment:
Unneeded, see: https://github.com/pagopa/interop-be-monorepo/pull/2407#discussion_r2378242011
```suggestion
```